LiteFarmOrg/LiteFarm
# Lf 2322 modify survey stack export document

This PR exports the answers to the questions asked during a certification document as an Excel sheet.

To Test:
First we need to set up the database to pull the correct surveys. In the psql client...
1. Connect to the 'pg-litefarm' database
2. run the following command `update certifiers set survey_id='627bdbff5b5f71000118f47c'`. This will update the surveys for all certifiers

Then we need to set up the Redis service

1. Under packages/api, you have to create a directory called exports
2. Install the aws cli https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
3. Authenticate the aws cli by running aws configure in your terminal, and using the DO_SPACES_ACCES_KEY_ID and DO_SPACES_SECRET_ACCESS_KEY values from the packages/api/.env file for the access key id and secret access key fields

Then start the service in a new terminal by typing `npm run scheduler` in the api folder

Then on localhost...
1. Navigate to the certifications page and select any certifier
4. Continue through the flow and fill in the survey
5. After completing the flow, check your email for the documents. You should see an Excel sheet with all of your answers in a format that fits the question
